### ISSUE TITLE
Too few point clouds are generated

### ISSUE THREAD

**Issue by kirikunkun** [graphdeco-inria/gaussian-splatting#1047]
![微信图片_20241104172706](https://github.com/user-attachments/assets/b2d3f80f-4280-48d4-bb40-2b8f3c755d7e)
![微信图片_20241104172718](https://github.com/user-attachments/assets/83e83c03-3991-4858-9498-891e5b56c709)
![微信图片_20241104172728](https://github.com/user-attachments/assets/261458de-433d-4482-8411-21c7292a347d)
![微信图片_20241104172838](https://github.com/user-attachments/assets/871c97aa-8056-4a3e-977f-9e1bd51038bd)
I used exactly the same settings as on the website and trained on the provided dataset. However, the training results were very poor. The point cloud file is much smaller than the pretrained model, and the rendered output is just a blank white image. Could someone help me understand the reason for this?

**Comment by kirikunkun** [comment]
![微信图片_20241104173448](https://github.com/user-attachments/assets/7fdf0771-518e-4f64-8084-ea5615d6d25b)
I found an error in this line in gaussian_model.py. Could this affect the rendering results?

**Comment by AsherJingkongChen** [comment]
> ![微信图片_20241104173448](https://github.com/user-attachments/assets/7fdf0771-518e-4f64-8084-ea5615d6d25b)
> I found an error in this line in gaussian_model.py. Could this affect the rendering results?

There should be a compilation error if distCUDA2 was not found. It should not matter because you did train.

**Comment by AsherJingkongChen** [comment]
@kirikunkun You scaling modifier is so small. Could it be the root cause? I see the default is 1.0.

You can also try to re-download the input file.

**Comment by kirikunkun** [comment]
> @kirikunkun You scaling modifier is so small. Could it be the root cause? I see the default is 1.0.
> 
> You can also try to re-download the input file.
Thank you for your response. I have tried adjusting the scaling modifier parameter, but the results are still unsatisfactory. The generated point cloud file is very small, an order of magnitude smaller than the pretrained model. For example, in this case, the pretrained model's point cloud file is several hundred thousand KB, whereas mine is only tens of thousands. The same issue has occurred on my friend's computer and has been reported by others as well, but we don't know the exact cause.

**Comment by kirikunkun** [comment]
> @kirikunkun You scaling modifier is so small. Could it be the root cause? I see the default is 1.0.
> 
> You can also try to re-download the input [file.]

(https://github.com/graphdeco-inria/gaussian-splatting/issues/1013#issue-2585908981)the same issue

**Comment by kirikunkun** [comment]
I resolved the issue. I followed the instructions on Colab to clone from Git, which also cloned the folders in the submodules. Previously, I couldn’t clone two files in the submodules, so I downloaded them using links from the comments. Now, I suspect that the issue might have been caused by incompatible versions of the submodules.
![微信图片_20241106154749](https://github.com/user-attachments/assets/c3a8113f-fae5-419a-821f-0bc701b2bee7)
